Source/santactl/Commands/SNTCommandFileInfo.m

```diff
@@ -13,7 +13,9 @@
 ///    limitations under the License.
 
 @import Foundation;
+@import AppKit;
```

Comment:
Using it for NSWorkspace to determine when a directory is a bundle before the SNTFileInfo object is created.